site/docs/v1/tech/apis/_webauthn-register-start-response-body.adoc

```diff
@@ -0,0 +1,89 @@
+==== Response Body
+
+[.api]
+[field]#options# [type]#[Object]#::
+The WebAuthn public key credential creation options. This object is passed to the WebAuthn client API to create a new passkey.
```

Comment:
"WebAuthn client API" is the javascript API, correct? Maybe link to https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API wherever this is mentioned?